kube/crawler.secrets.yaml.template

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
This is the file I think it can be deleted. This still could be part of the documentation though!